Directory.Build.targets

```diff
@@ -23,6 +23,8 @@
   <PropertyGroup Condition=" '$(MSBuildProjectExtension)' == '.csproj' ">
     <PackageId Condition=" '$(PackageId)' == '' ">$(AssemblyName)</PackageId>
     <IsPackable Condition="'$(IsPackable)' == '' AND ( '$(IsTestProject)' == 'true' OR '$(IsTestAssetProject)' == 'true' OR '$(IsBenchmarkProject)' == 'true' OR '$(IsSampleProject)' == 'true' ) ">false</IsPackable>
+    <HasReferenceAssembly Condition="'$(IsPackable)' == 'false' OR '$(IsShipping)' == 'false'">false</HasReferenceAssembly>
```

Comment:
This repo, unlike aspnet/Extensions, has assemblies which are not in packages which should still have ref assemblies. I think this should be `Condition="'$(IsAspNetCoreApp)' == 'true'"`